proposals/field-keyword.md

```diff
@@ -356,33 +356,35 @@ primary_no_array_creation_expression
-> property with ~~semicolon-only accessor bodies. Auto-properties must have a get accessor and can optionally~~
-> ~~have a set accessor.~~ **either or both of:**
+> An automatically implemented property (or auto-property for short), is a non-abstract, non-extern, non-ref-valued
+> property with ~~semicolon-only accessor bodies. Auto-properties shall have a get accessor and may optionally have a set accessor.~~ **either or both of:**
 > 1. **an accessor with a semicolon-only body**
 > 2. **usage of the `field` contextual keyword within the accessors or**
 >    **expression body of the property**
 > 
-> When a property is specified as an ~~automatically implemented property~~ **auto-property**, a hidden **unnamed** backing field is automatically
+> When a property is specified as an automatically implemented property, a hidden **unnamed** backing field is automatically
 > available for the property ~~, and the accessors are implemented to read from and write to that backing field~~.
 > **For auto-properties, any semicolon-only `get` accessor is implemented to read from, and any semicolon-only**
 > **`set` accessor to write to its backing field.**
 > 
+> ~~The hidden backing field is inaccessible, it can be read and written only through the automatically implemented property accessors, even within the containing type.~~
 > **The backing field can be referenced directly using the `field` keyword**
 > **within all accessors and within the property expression body. Because the field is unnamed, it cannot be used in a**
 > **`nameof` expression.**
 > 
-> If the auto-property has ~~no set accessor~~ **only a semicolon-only get accessor**, the backing field is considered `readonly` ([§14.5.3](https://github.com/dotnet/csharpstandard/blob/standard-v6/standard/classes.md#1453-readonly-fields)).
-> Just like a `readonly` field, a getter-only auto property **(without a set accessor or an init accessor)** can also be assigned to in the body of a constructor
-> of the enclosing class. Such an assignment assigns directly to the ~~readonly~~ backing field of the property.
+> If the auto-property has ~~no set accessor~~ **only a semicolon-only get accessor**, the backing field is considered `readonly` ([§15.5.3](https://github.com/dotnet/csharpstandard/blob/standard-v7/standard/classes.md#1553-readonly-fields)).
+> Just like a `readonly` field, a read-only auto-property **(without a set accessor or an init accessor)** may also be assigned to in the body of a constructor
+> of the enclosing class. Such an assignment assigns directly to the ~~read-only~~ backing field of the property.
 > 
 > **An auto-property is not allowed to only have a single semicolon-only `set` accessor without a `get` accessor.**
+> 
+> An auto-property may optionally have a *property_initializer*, which is applied directly to the backing field as a *variable_initializer* ([§17.7](https://github.com/dotnet/csharpstandard/blob/standard-v7/standard/arrays.md#177-array-initializers)).
```

Comment:
I wonder if the spec meant to link to <https://github.com/dotnet/csharpstandard/blob/standard-v7/standard/classes.md#1556-variable-initializers> instead of linking to array initializers specifically.

Reply:
You may be right. I'll leave the link as-is though, to minimize the changes from the spec.

Reply:
Filed https://github.com/dotnet/csharpstandard/issues/1163.